Kickstarter-iOS/DataSources/ShippingRulesDataSource.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Library
+import UIKit
+
+final class ShippingRulesDataSource: ValueCellDataSource {
+  // MARK: - Load
+
+  func load(_ values: [String]) {
```

Comment:
We shouldn't really need the `ShippingRule` type as we can always find that during selection by the index of selected row (this will be done in the follow up work).